### ISSUE TITLE
Can I "borrow" some of this into flake-firefox-nightly?

### ISSUE THREAD

**Issue by colemickens** [calbrecht/nixpkgs-overlays#3]
Hi @calbrecht,

I have recently been working on `flake-firefox-nightly`. It's a layer on top of [nixpkgs-mozilla](https://github.com/mozilla/nixpkgs-mozilla) that pins the firefox derivations to specific versions.

My next remaining thing for Firefox is to get this pipewire scenario working, and the solution I'm inclined to use is to make it available via `flake-firefox-nightly`. Is it okay if I end up copying patches from here to that repo, etc? I'll happily point back here, link/credit you, etc.

**Comment by colemickens** [comment]
Do you still need a non-released version of pipewire? Do you know what commits you needed from master?

**Comment by calbrecht** [comment]
@colemickens of course, copy whatever you like, you're welcome. A link back is welcome, too :+1:
 
The pipewire version from nixpkgs master/unstable should be fine too (fedora patch is still the same since two months and targets the 0.3 version, see [firefox-pipewire-0-3.patch](https://src.fedoraproject.org/rpms/firefox/history/firefox-pipewire-0-3.patch?identifier=master) history at fedora (i think they are using the "official" pipewire release for that)).

It's just me wanting the latest stuff, i guess.
